build/Targets/Settings.props

```diff
@@ -239,6 +239,7 @@
     <DefineConstants Condition="'$(InitialDefineConstants)' != ''">$(InitialDefineConstants)</DefineConstants>
 
     <!-- The NoWarn property is still being unconditionally set.  https://github.com/dotnet/sdk/issues/752 -->
-    <NoWarn>$(NoWarn);1591</NoWarn>
+    <OptionStrict>Off</OptionStrict>
```

Comment:
Should be `<OptionStrict>On</OptionStrict>`.   See

https://github.com/dotnet/roslyn/blob/master/build/Targets/Settings.props#L131